src/about.njk

```diff
@@ -79,17 +79,17 @@
         <p class="text-muted">Knoxville leader</p>
         <ul class="list-inline social-buttons">
           <li class="list-inline-item">
-            <a href="https://github.com/stefanhk31" target="_blank" class="shadow-sm">
+            <a href="https://github.com/stefanhk31" target="_blank" class="shadow-sm" aria-label="Navigation link to Stefan Hodges-Kluck GitHub Page">
```

Comment:
Screen readers will announce this as a link, so a better aria-label would be "Stefan Hodges-Kluck's GitHub Page"

Reply:
fixed